docs/hacktoberfest/index.html

```diff
@@ -1,199 +1,41 @@
-        <div id="templates">
-            <!-- TEMPLATE DE UMA ATIVIDADE -->
-            <div class="activity" id="activity-template">
-                <div class="start-time">
-                  <p class="hour"></p>
-                  <p class="minutes"></p>
-                </div>
-                <div class="details">
-                  <h3 class="title"></h3>
-                  <p class="speaker"></p>
-                </div>
-              </div>
-              
-                <!-- TEMPLATE DE UMA PAUSA-->
-              <div class="activity break" id="break-template">
-                  <div class="start-time">
-                    <span class="hour"></span>
-                    <span>:</span>
-                    <span class="minutes"></span>
-                    <h3 class="title"></h3>
-                  </div>
-              </div>
+            
+        <div id="timer-section">
+            <div id="timer">
+                <div id="days"></div> <span class="time-label">d</span> &nbsp
+                <div id="hours"></div> <span class="time-label">h</span> &nbsp
+                <div id="minutes"></div> <span class="time-label">m</span> &nbsp
+                <div id="seconds"></div> <span class="time-label">s</span> &nbsp
+            </div>
         </div>
-      </main>
-    </section>
-    <!-- SESSÃO LOCALIZAÇÃO  -->
-    <section id="localization">
-      <header class="section-title">
-        <h3>LOCALIZACAO</h3>
-      </header>
-      <main>
-        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1979.1229886131034!2d-35.90863294212522!3d-7.212756698698225!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x7ac1fcd9e578cbb%3A0x9390eee9ee6305a7!2sBloco%20CP1%20-%20Edif%C3%ADcio%20Professor%20Camilo%20de%20Lelis%20Gondim%20Medeiros!5e0!3m2!1spt-BR!2sbr!4v1570821382827!5m2!1spt-BR!2sbr" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
-      </main>
+      
     </section>
-    <!-- FOOTER -->
-    <footer>
-      <div id="footer-content">
-        <div id="footer-logo">
-          <a href="https://opendevufcg.org/">
-            <img src="./assets/img/logo.svg">
-          </a>
-        </div>
-        <div id="sponsors">
-          <p>PATROCINADORES</p>
-          <div>
-            <a href="http://www.computacao.ufcg.edu.br" target="_blank">
-              <img src="./assets/img/computacao.svg" alt="icone computação">
-            </a>
-            <a href="https://vtex.com/br-pt/" target="_blank">
-              <img src="./assets/img/vtex.svg" alt="icone vtex">
-            </a>
-          </div>
-        </div>
-      </div>
-      <p id="copyright">© Opendev 2019 - Todos os direitos reservados. </p>
-    </footer>
-    <script src="data/atividades.js"></script>
-    <script src="js/counter.js"></script>
-    <script src="js/app.js"></script>
-    <script src="../sw-register.js"></script>
-  </body>
-</html>
\ No newline at end of file
+    <script src="./js/timer.js"></script> 
```

Comment:
Ao invés de colocar essa tag no fim, imagino que o que você queira fazer com isso, é só carregar o script do timer após todo o html ter sido carregado, tem uma forma de fazer isso em JavaScript com `defer`
https://bitsofco.de/async-vs-defer/
```suggestion
    <script defer src="./js/timer.js"></script>
```
Aí essa declaração poderia ir pro topo.
Mas caso não seja o que pensei, pode desconsiderar :D

Reply:
Uma vez que usando defer ele vai carregar não depois do html ser renderizado não importa onde ele seja carregado, tem algum motivo de importar no topo ou logo antes de fechar a tag body?

Reply:
Isso, o lugar não tem importância, pode ser tanto no topo, quanto embaixo, já que o `defer` vai cuidar do efeito de execução pode deixar ai tb.